src/Simplex_tree/test/CMakeLists.txt

```diff
@@ -40,3 +40,9 @@ if (TBB_FOUND)
   target_link_libraries(Simplex_tree_graph_expansion_test_unit ${TBB_LIBRARIES})
 endif()
 gudhi_add_boost_test(Simplex_tree_graph_expansion_test_unit)
+
+add_executable ( Simplex_tree_serialization_test_unit simplex_tree_serialization_unit_test.cpp )
+if (TBB_FOUND)
+  target_link_libraries(Simplex_tree_serialization_test_unit ${TBB_LIBRARIES})
```

Comment:
Do we use the functions that use TBB in this test? Well, at worst it links unnecessarily, that does hurt, so ok.

Reply:
Ok, removed on [a528405](https://github.com/GUDHI/gudhi-devel/pull/810/commits/a528405db845923996d284367d3b1c4afbbb1e31)